src/js/claims-status/containers/YourClaimsPageV2.jsx

```diff
@@ -188,6 +188,14 @@ class YourClaimsPageV2 extends React.Component {
           <div className="small-12 usa-width-one-third medium-4 columns help-sidebar">
             <FeaturesWarning/>
             <AskVAQuestions/>
+            <div>
```

Comment:
Should we put this into a component similar to AskVAQuestions? Would that enable us to reuse some of that components styles and allow us to avoid creating new style rules?

Reply:
I don't know what AskVAQuestions is but assuming that it means a forum where we can talk about site wide things, I think it is a great idea. My guess is also that after font Bitter was on vetsdotgov, a lot of the spacing shifted.

Reply:
@gnakm `AskVAQuestions` is a reusable component that displays the "Need help?" section of the sidebar on this page.

@cehsu I thought about it, but it didn't seem like a great idea. `AskVAQuestions` is used on a number of different pages in the claims status app, but this "Can't find your appeals?" bit is only used here. Abstracting it out for the two sections to share a common component seemed overkill, and creating a new component that's only used here seemed like more overhead than it was worth.

As far as the new style rule goes, that was needed for the spacing between the sidebar components which wasn't there in the first place, so we'd need to make them either way.